src/components/NotFound/NotFound.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import EagleAnimation from './EagleAnimation'
+import { theme, breakpoint, Button } from '@aragon/ui'
+const medium = css => breakpoint('medium', css)
+
+class NotFound extends React.Component {
+  state = { details: false }
+  clickMoreDetails = () => {
+    const details = this.state.details
+    this.setState({ details: !details })
+  }
+  render() {
+    const {
+      title,
+      issue,
+      reload,
+      detailsTitle,
+      detailsContent,
+      children,
+    } = this.props
+    const { details } = this.state
+    return (
+      <div>
+        <EagleAnimation />
+        <Card>
+          <h1>{title}</h1>
+          {children}
+          {(detailsContent || detailsTitle) && (
+            <div>
+              <DetailsButton onClick={this.clickMoreDetails}>
+                {details ? 'Hide details...' : 'More details...'}
+              </DetailsButton>
+              {details && (
+                <DetailsContainer>
+                  {detailsTitle && <h2>{detailsTitle}</h2>}
+                  {detailsContent}
+                </DetailsContainer>
+              )}
+            </div>
+          )}
+          {(issue || reload) && (
+            <ButtonBox>
+              {issue && (
+                <IssueLink>
+                  <Button.Anchor
+                    mode="text"
+                    href="https://github.com/aragon/aragon/issues"
+                    target="_blank"
+                  >
+                    Tell us what went wrong
+                  </Button.Anchor>
+                </IssueLink>
+              )}
+              {reload && (
+                <Button mode="strong" onClick={() => location.reload()}>
+                  Reload
+                </Button>
+              )}
+            </ButtonBox>
+          )}
+        </Card>
+      </div>
+    )
+  }
+}
+
+const Card = styled.div`
+  background: white;
+  -webkit-box-shadow: 0px 0px 19px 0px #c9c9c9;
+  -moz-box-shadow: 0px 0px 19px 0px #c9c9c9;
+  box-shadow: 0px 0px 19px 0px #c9c9c9;
+  padding: 30px;
+  border: solid 1px #c9c9c9;
+  border-radius: 4px;
+  max-width: 550px;
+  color: ${theme.textSecondary};
+  margin: auto 15px;
+  margin-top: -90px;
+  z-index: 2;
+  position: relative;
+  a {
+    cursor: pointer;
+    color: ${theme.accent};
+  }
+  a:hover,
+  a:active {
+    color: ${theme.gradientStartActive};
+  }
+  h1 {
+    color: ${theme.textDimmed};
+    font-size: 20px;
+    margin-bottom: 20px;
+    ${medium('font-size: 30px;')};
+  }
+`
+
+const DetailsButton = styled.p`
```

Comment:
Maybe we could use `Button` here? That way we’ll get focus state (when implemented) and keyboard navigation.